.eslintrc.js

```diff
@@ -240,6 +239,92 @@ module.exports = {
+    'unicorn/no-await-expression-member': 2,
+    'unicorn/no-console-spaces': 2,
+    'unicorn/no-document-cookie': 2,
+    'unicorn/no-empty-file': 2,
+    'unicorn/no-for-loop': 2,
+    'unicorn/no-hex-escape': 2,
+    'unicorn/no-instanceof-array': 2,
+    'unicorn/no-invalid-remove-event-listener': 2,
+    'unicorn/no-keyword-prefix': 0,
+    'unicorn/no-lonely-if': 2,
+    'unicorn/no-nested-ternary': 2,
+    'unicorn/no-new-array': 2,
+    'unicorn/no-new-buffer': 2,
+    'unicorn/no-object-as-default-parameter': 2,
+    'unicorn/no-static-only-class': 2,
+    'unicorn/no-thenable': 2,
+    'unicorn/no-this-assignment': 2,
+    'unicorn/no-unsafe-regex': 0,
+    'unicorn/no-useless-fallback-in-spread': 2,
+    'unicorn/no-useless-length-check': 2,
+    'unicorn/no-useless-promise-resolve-reject': 2,
+    'unicorn/no-useless-spread': 2,
+    'unicorn/no-useless-undefined': 2,
+    'unicorn/no-zero-fractions': 2,
+    'unicorn/numeric-separators-style': 2,
+    'unicorn/prefer-add-event-listener': 2,
+    'unicorn/prefer-array-find': 2,
+    'unicorn/prefer-array-flat': 2,
+    'unicorn/prefer-array-flat-map': 2,
+    'unicorn/prefer-array-index-of': 2,
+    'unicorn/prefer-array-some': 2,
+    // TODO: Enable this by default when targeting a Node.js version that supports `Array#at`.
+    'unicorn/prefer-at': 0,
+    'unicorn/prefer-code-point': 2,
+    'unicorn/prefer-date-now': 2,
+    'unicorn/prefer-default-parameters': 2,
+    'unicorn/prefer-dom-node-append': 2,
+    'unicorn/prefer-dom-node-dataset': 2,
+    'unicorn/prefer-dom-node-remove': 2,
+    'unicorn/prefer-dom-node-text-content': 2,
+    'unicorn/prefer-export-from': 2,
+    'unicorn/prefer-includes': 2,
+    'unicorn/prefer-json-parse-buffer': 2,
+    'unicorn/prefer-keyboard-event-key': 2,
+    'unicorn/prefer-math-trunc': 2,
+    'unicorn/prefer-modern-dom-apis': 2,
+    'unicorn/prefer-negative-index': 2,
+    'unicorn/prefer-number-properties': 2,
+    'unicorn/prefer-optional-catch-binding': 2,
+    'unicorn/prefer-prototype-methods': 2,
+    'unicorn/prefer-query-selector': 2,
+    'unicorn/prefer-reflect-apply': 2,
+    'unicorn/prefer-regexp-test': 2,
+    'unicorn/prefer-set-has': 2,
+    'unicorn/prefer-spread': 2,
+    // TODO: Enable this by default when targeting Node.js 16.
+    'unicorn/prefer-string-replace-all': 0,
+    'unicorn/prefer-string-slice': 2,
+    'unicorn/prefer-string-starts-ends-with': 2,
+    'unicorn/prefer-string-trim-start-end': 2,
+    'unicorn/prefer-switch': 2,
+    'unicorn/prefer-ternary': 2,
+    // TODO: Enable this by default when targeting Node.js 14.
+    'unicorn/prefer-top-level-await': 0,
+    'unicorn/prefer-type-error': 2,
+    'unicorn/relative-url-style': 2,
+    'unicorn/require-array-join-separator': 2,
+    'unicorn/require-number-to-fixed-digits-argument': 2,
+    // Turned off because we can't distinguish `window.postMessage` and `{Worker,MessagePort,Client,BroadcastChannel}#postMessage()`
+    // See #1396
```

Comment:
Is this a reference to an issue on the unicorn repo?

Reply:
Yes! I just copy/pasted it :man_facepalming:
Fixed :+1: